src/lancie-admin-app.html

```diff
@@ -18,17 +20,64 @@
         @apply(--layout-flex);
       }
     </style>
-    <lancie-admin-layout class="layout vertical" route="{{route}}">
-      <lancie-admin-content page="{{route}}"></lancie-admin-content>
-    </lancie-admin-layout>
-  </template>
 
+    <template is="dom-if" if="[[loggedIn]]">
+      <lancie-admin-layout class="layout vertical" route="{{route}}">
+        <lancie-admin-content page="{{route}}"></lancie-admin-content>
+      </lancie-admin-layout>
+    </template>
+
+    <template is="dom-if" if="[[!loggedIn]]">
+      <lancie-login-card on-logged-in="storeLogin"></lancie-login-card>
+    </template>
+
+    <lancie-ajax id="ajax"></lancie-ajax>
+    <iron-ajax
+      id="getCurrentUser"
+      url="/api/v1/users/current"
+      on-response="onCurrentUser"></iron-ajax>
+
+  </template>
   <script>
   (function() {
     'use strict';
 
     Polymer({
       is: 'lancie-admin-app',
+      properties: {
+        user: Object,
+        token: String,
+        loggedIn: {
+          type: Boolean,
+          value: false,
+        },
+      },
+
+      attached: function() {
+        var token = window.sessionStorage.getItem('auth-token');
```

Comment:
Good point, I will change the areafiftylan frontend over to the localstorage soon. This implementation is easier in my opinion.